measuring_federate/server.py

```diff
@@ -26,17 +37,20 @@ async def read_root():
     ).dict()
     return JSONResponse(response, 200)
     
-@app.post("/run/")
+@app.post("/run")
 async def run_model(broker_config:BrokerConfig, background_tasks: BackgroundTasks):
-    try:
-        print(broker_config)
-        feeder_ip = broker_config.services['oedisi_feeder']['networks']['custom-network']['ipv4_address']
-        feeder_port = int(broker_config.services['oedisi_feeder']['ports'][0].split(":")[0])
-       
-        url =f"http://{feeder_ip}:{feeder_port}/sensor/"
-        print(url)
+    logging.info(broker_config)
+    feeder_host = broker_config.services['oedisi_feeder']['hostname']
+    feeder_port = int(broker_config.services['oedisi_feeder']['ports'][0].split(":")[0])
+    url = build_url(feeder_host, feeder_port, 'sensor') 
+    logging.info(url)
+    try:   
         reply = requests.get(url)
         sensor_data = reply.json()
+        if not sensor_data:
+            msg = "empty sensor list"
+            raise HTTPException(404, msg)
+        logging.info(sensor_data)
         with open("sensors.json", "w") as outfile:
```

Comment:
When we use smart-ds, we use the create_measurement_lists function to create voltage_ids.json, real_ids.json, and reactive_ids.json, but not a sensors.json.

Reply:
will have to resolve this issue once we have sensors published on the helics interface